<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File System Utils</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblRuta">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Direccion-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtras">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdelante">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
    <property name="icon">
     <iconset resource="folder.qrc">
      <normaloff>:/jixzMaA5T.png</normaloff>:/jixzMaA5T.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPegar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pegar</string>
    </property>
    <property name="icon">
     <iconset resource="folder.qrc">
      <normaloff>:/Imagenes/thin-0255_paste_clipboard-512.png</normaloff>:/Imagenes/thin-0255_paste_clipboard-512.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNuevo">
    <property name="title">
     <string>Nuevo</string>
    </property>
    <addaction name="actionCarpeta"/>
    <addaction name="actionArchivo"/>
   </widget>
   <addaction name="menuNuevo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCarpeta">
   <property name="icon">
    <iconset resource="folder.qrc">
     <normaloff>:/Imagenes/Folders-PNG-File.png</normaloff>:/Imagenes/Folders-PNG-File.png</iconset>
   </property>
   <property name="text">
    <string>Carpeta</string>
   </property>
  </action>
  <action name="actionArchivo">
   <property name="icon">
    <iconset resource="folder.qrc">
     <normaloff>:/Imagenes/Docs-icon-iloveimg-resized.png</normaloff>:/Imagenes/Docs-icon-iloveimg-resized.png</iconset>
   </property>
   <property name="text">
    <string>Archivo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="folder.qrc"/>
 </resources>
 <connections/>
</ui>
